server/data/nomisUserRolesApi.ts

```diff
@@ -0,0 +1,35 @@
+import config from '../config'
+import RestClient from './restClient'
+
+interface Caseload {
+  id: string
+  name: string
+}
+
+interface UserCaseload {
+  activeCaseload: Caseload
+  caseloads: Array<Caseload>
+}
+
+class NomisUserRolesApi extends RestClient {
+  constructor(token: string) {
+    super('NOMIS User Roles API', config.apis.nomisUserRolesApi, token)
+  }
+
+  async getUserCaseloads(): Promise<UserCaseload> {
+    const userCaseload = (await this.get({ path: '/me/caseloads' })) as UserCaseload
+
+    const caseloads =
+      userCaseload.caseloads != null
+        ? userCaseload.caseloads.filter(caseload => {
+            return caseload.id !== 'NWEB'
```

Comment:
What's special about this `NWEB` prison?

Reply:
NWEB is a special caseload which is being returned. But I've remove it from the results in this PR https://github.com/ministryofjustice/nomis-user-roles-api/pull/94